core/embed/rust/src/ui/model_t1/layout.rs

```diff
@@ -60,12 +58,7 @@ extern "C" fn ui_layout_new_confirm_action(
     unsafe { util::try_with_args_and_kwargs(n_args, args, kwargs, block) }
 }
 
-#[no_mangle]
-extern "C" fn ui_layout_new_confirm_text(
-    n_args: usize,
-    args: *const Obj,
-    kwargs: *const Map,
-) -> Obj {
+extern "C" fn new_confirm_text(n_args: usize, args: *const Obj, kwargs: *mut Map) -> Obj {
```

Comment:
Note that I changed `*const Map` to `*mut Map` to pass type check.

Reply:
I can't check it now, but it should match the type the C-side expects.